### DISCUSSION TITLE
2022-11-02 Meeting Agenda, Recording & Notes

### DISCUSSION THREAD

**Discussion by andy-blum** [ddev/ddev#4191] [DDEV Advisory Group]
Previous meeting: #3963
Next meeting: Jan 4, 2023

## Agenda

* Turn on recording, ask for a minutes-taker
* Progress in last 2 months
* Introduce docs-driver Matt Stein, 5-minutes on what he's up to
* Proposal: Yearly review and planning for January meeting.
* "Bus factor" updates, shared passwords, etc.
* Upstream donations status
* Financial status/reports
* Anything on anybody's mind!

## Meeting Notes

https://docs.google.com/document/d/1VU2dHNi22RjINGtI3fy1gAOb2dTLdNOFqLzVV0OBv9Q/edit?usp=sharing

## Recording

https://www.dropbox.com/s/tqi6er65ije4dtx/20221102%20DDEV%20Advisory%20Group%20meeting.mp4?dl=0

## Financial report

[Localdev Foundation Cash Flow 2022-11-01-21_57.pdf](https://github.com/drud/ddev/files/9914241/Localdev.Foundation.Cash.Flow.2022-11-01-21_57.pdf)

**Comment by chrisfromredfin** [comment]
NOTES: https://docs.google.com/document/d/1VU2dHNi22RjINGtI3fy1gAOb2dTLdNOFqLzVV0OBv9Q/edit?usp=sharing

**Reply by rfay** [reply]
Added recording above, thanks all. Thanks for the notes and caretaking here @chrisfromredfin !
